calculator.c

```diff
@@ -0,0 +1,38 @@
+//Bettina Benitez
```

Comment:
Your code didn't handle floats like it's supposed to. Additionally, you didn't implement the mod function.